README.md

```diff
@@ -195,6 +195,62 @@ var stringElement = minim.toElement("foobar");
 var stringElementClone = stringElement.clone();
 ```
 
+#### findRecursive
+
+Recursively find an element. Returns an ArrayElement containing all elements
+that match the given element name.
+
+```javascript
+const strings = element.findRecursive('string');
+```
+
+You may pass multiple element names to `findRecursive`. When multiple element
+names are passed down, minim will only find the an element that is found within
+the other given elements. For example, we can pass in `member` and `string` so
+that we are recursively looking for all `string` elements that are found within a
+`member` element:
+
+```javascript
+const stringInsideMembers = element.findRecursive('member', 'string');
+```
+
+##### Parents
+
+Each returned element will include a new `parents` property which is an array
+element including the parents of the returned element.
+
+As an example, if I had an array element which contains a category element
+which in turn contains a string element with the content "Hello World". I can
+access the parent array and category element.
+
+```json
+{
+  "element": "array",
+  "content": [
+    {
+      "element": "category",
+      "content": [
+        {
+          "element": "string",
+          "content": "Hello World"
+        }
+      ]
+    }
+  ]
+}
+```
+
+```javascript
+const elements = element.findRecursive('string');
+const helloString = elements.first();
+
+// Array Element
+helloString.parents[0];
+
+// Category Element
+helloString.parents[1];
+```
```

Comment:
I think the enumeration would make more sense reversed. If you read it out loud: "category is the 1st parent of helloString", "array is the 2nd parent of helloString"